### PR TITLE
use relatedVenues for back space dropdown

### PR DESCRIPTION
**Pull request by denisdimitrov**
`relatedVenues` are now used for the back to space dropdown, instead of `ownedVenues`

Fixes: https://github.com/sparkletown/internal-sparkle-issues/issues/1685#issuecomment-1041055535

### CODE REVIEW

**Comment by denisdimitrov** [comment]
My precommit linter ran and found no problems, yet the CI tells otherwise. Any ideas? @sparkletown/engineering

**Review by goran-peoski-work — APPROVED** [review]
LGTM

**Comment by goran-peoski-work** [comment]
@denisdimitrov @whoaitsaimz the issue
- https://github.com/sparkletown/internal-sparkle-issues/issues/1685#issuecomment-1041055535
is moved to DONE

Is this PR still necessary?

File: src/components/molecules/SpaceEditForm/SpaceEditForm.tsx [hunk 1]
```diff
@@ -353,26 +353,26 @@ export const SpaceEditForm: React.FC<SpaceEditFormProps> = ({ space }) => {
...
-                  <div>
-                    The preset map backgrounds could not be fetched. Please,
-                    refresh the page or upload a custom map background.
-                  </div>
-                  <div>Error: {errorFetchBackgrounds.message}</div>
-                </>
-              )}
-            </AdminSidebarAccordion>
-          )}
+              <AdminSidebarAccordion title="Map background">
```
Comment (goran-peoski-work) [inline src/components/molecules/SpaceEditForm/SpaceEditForm.tsx:356]: [Q] I don't notice code change, how does this happen? Isn't reformat on commit supposed to produce the same indent?

**Review by goran-peoski-work — APPROVED** [review]
LGTM

**Comment by denisdimitrov** [comment]
Stale and outdated. Old design and task.

**Pull request closed without merging**